O365/account.py

```diff
@@ -36,8 +36,10 @@ def __init__(self, credentials, *, protocol=None, main_resource=None, **kwargs):
         elif auth_flow_type == 'credentials':
             # for client credential grant flow solely:
             # append the default scope if it's not provided
-            if not scopes:
-                kwargs['scopes'] = self.protocol.prefix_scope('.default')
+            if scopes:
+                kwargs['scopes'] = self.protocol.get_scopes_for(scopes)
+                if '.default' not in scopes:
```

Comment:
You can't rely on this. What if the user passed 'https://graph.microsoft.com/.default' to scopes?
Will then will add the .default scope twice.